style.css

```diff
@@ -580,7 +580,7 @@ q:after {
 
 /* Typography for Thai Font */
 
-html[lang="th"] h1, 
+html[lang="th"] h1,
```

Comment:
Unrelated to the fix, but that's okay - it looks like your editor cleaned up a stray extra space 🙂